HACKING.md

```diff
@@ -0,0 +1,88 @@
+# Hacking
+
+## `make-prs.py`
+
+The `make-prs.py` script (only exists in the main branch) does the following:
+
+- Get the diff from a PR;
+- find out which chapter that PR is about;
+- create branches (in your forked repo), merge the diff into all following chapters, commit, and push;
+- create PRs to the upstream.
+
+### Configuration
+
+- GITHUB_TOKEN env var is set.
+
+### Dependencies
+
+- `git` CLI
+- `tox`
+- `PyGithub`, `requests` (both are already automatically managed by `tox`)
```

Comment:
I don't think we need to mention these here, as they're installed and handled automatically by tox.